notebooks/interfield_differences.ipynb

```diff
@@ -0,0 +1,1245 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.            # If the node is unshuffleable for some reason, its percentile isn't meaningful
</pre><p>What does "unshuffleable" mean? It wasn't included in any of the shuffled networks, or its edges weren't able to be permuted, or something else?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Right, if the node has zero edges or is in a connected component that is too small it may not be able to be shuffled. I don't think this case actually catches anything, I added it to see if it cleared up the phenomenon where all the low-pagerank articles have high percentiles</p>